program/src/instruction.rs

```diff
@@ -3114,6 +3114,25 @@ pub fn liquidate_delisting_token(
     Ok(Instruction { program_id: *program_id, accounts, data })
 }
 
+pub fn create_dust_account(
+    program_id: &Pubkey,
+    mango_group_pk: &Pubkey,
+    mango_account_pk: &Pubkey,
+    payer_pk: &Pubkey,
+) -> Result<Instruction, ProgramError> {
+    let accounts = vec![
+        AccountMeta::new_readonly(*mango_group_pk, false),
+        AccountMeta::new(*mango_account_pk, false),
```

Comment:
it's not related to this PR, but I find it suspicious that `create_dust_account()` allows creating a dust account at an arbitrary address. As far as I can tell the dust account has to be at the seed `[mango_group, "DustAccount"]`

Reply:
I believe `seed_and_create_pda` checks that the passed address matches that generated by the seeds https://github.com/blockworks-foundation/mango-v3/blob/main/program/src/processor.rs#L7544